modules/nw-ovn-kubernetes-ansible-migration-about.adoc

```diff
@@ -0,0 +1,118 @@
+
+The Ansible playbook includes the following set of playbooks that each represent a stage in the migration process:
+
+. Prechecks
+. Migrate
+. Reboot
+. Post-migration
+
+[IMPORTANT]
+====
+If you run a playbook as a non-cluster administrator, the playbook fails.
+====
+
+If a playbook fails, check the playbook output for the source of the error. Resolve the error and then rerun the Ansible playbook.
+
+.Prerequisites
+
+* You set a bastion server.
+// LINK for this packages in a RH resource?
+* You installed `ansible`, `jq`, `pip3`, and `python3` packages.
+** To install the `ansible` package, enter the `python3 -m pip install --user ansible` command.
+// SET TO WHAT?
+* You set the `KUBECONFIG` environment variable.
+* See "Migrating to the OVN-Kubernetes network plugin by using the offline migration method" for a list of other prerequisites.
+
+.Procedure
+
+// Is this a safe resource
+. Clone the link:https://github.com/openshift/ansible-ocp-networking-migration-rollback[`ansible-sdn-to-ovn-migration.git`] repository to your local system.
+
+. Change to the directory for the cloned repository by entering the following command:
++
+[source,terminal]
+----
+$ cd ansible-sdn-to-ovn-migration.git
+----
+
+// jq correct here?
+. To start the Ansible playbook enter the following command in the `oc` CLI:
++
+[source,terminal]
+----
+$ ansible-playbook -v playbook-migration.yml
+----
+
+. Check the output from the Pre-check playbook confirms completion for each task. The playbook marks each completed task with `ok:`.
+
+. Check the output from the Migrate playbook confirms completion for each task. The playbook marks each completed task with either `changed:` or `ok:`.
+
+. List the machine configuration state and the name of the applied machine configuration by entering the following command: 
++
+[source,terminal]
+----
+$ oc describe node | egrep "hostname|machineconfig"
+----
++
+.Example output
+[source,terminal]
+----
+kubernetes.io/hostname=master-0
+machineconfiguration.openshift.io/currentConfig: rendered-master-c53e221d9d24e1c8bb6ee89dd3d8ad7b
+machineconfiguration.openshift.io/desiredConfig: rendered-master-c53e221d9d24e1c8bb6ee89dd3d8ad7b
+machineconfiguration.openshift.io/reason:
+machineconfiguration.openshift.io/state: Done
+----
+
+. Confirm that the machine config is correct by entering the following command:
++
+[source,terminal]
+----
+$ oc get machineconfig <config_name> -o yaml | grep ExecStart <1>
+----
+<1> Where `<config_name>` is the name of the machine config from the `machineconfiguration.openshift.io/currentConfig` field.
++
+The machine config must include the following update to the systemd configuration:
++
+[source,terminal]
+----
+ExecStart=/usr/local/bin/mtu-migration.sh
+----
+
+. Enter the following `patch` command to migrate from the OpenShift software-defined networking (SDN) plugin to the OVN-Kubernetes network plugin:
```

Comment:
this step is handled by the playbook and not required to be entered. cc @miheer

Reply:
Thanks. Commented out for now.